### PR TITLE
Add JS SDK as submodule

### PR DESCRIPTION
**Pull request by hackerwins**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[submodule "yorkie-js-sdk"]
+	path = yorkie-js-sdk
```
Comment (mojosoeun) [inline .gitmodules]: can we change this path from `yorkie-js-sdk` to `docs/js-references`?

Comment (hackerwins) [reply .gitmodules]: @mojosoeun I changed the path.

Comment (mojosoeun) [reply .gitmodules]: Thank you so much!

**Comment by hackerwins** [comment]
@mojosoeun Should this PR be merged before #36?

**Pull request closed without merging**